packages/react-imask/index.d.ts

```diff
@@ -0,0 +1,63 @@
+declare module "react-imask" {
+  import { Component, InputHTMLAttributes, Ref } from "react";
+
+  export type MaskType = string | typeof RegExp | Date | Number;
```

Comment:
The example value for `mask` is passing the prop `mask={Number}`. Does that mean this should be `typeof Number`? Same for Date (`typeof Date`) ?